recipe/meta.yaml

```diff
@@ -10,17 +10,18 @@ source:
   sha256: c540ea59173e0a291e19c742dd8b406c56e2be039a600edb7c6fc3ae4bbdfa9f
 
 build:
-  noarch: python
+  skip: True  # [py<37]
```

Comment:
It's not a good idea to change architecture without version bumping. In this case, we can build it as noarch